modules/ROOT/pages/comments-options.adoc

```diff
@@ -0,0 +1,12 @@
+:navtitle: Comments Options
+:description: TinyMCE Comments plugin options.
+:keywords: comments, commenting, tinycomments
+:pluginname: Comments
+:plugincode: comments
+:pluginminimumplan: enterprise
```

Comment:
should this be `essential`?
https://www.tiny.cloud/pricing/

Reply:
I think we should make a ticket to remove all references to this variable. We no longer include any plan details in the documentation so this variable is no longer used.